twitter-lm.css

```diff
@@ -0,0 +1,491 @@
+  height: 100%;
+  align-items: center;
+  cursor: pointer;
+}
+
+.tweet-options {
+  grid-auto-rows: 24px;
+  display: grid;
+  grid-auto-flow: column;
+  position: relative;
+}
+
+.flex {
+  display: flex;
+}
+
+.gap-sm {
+  gap : 5px;
+}
+
+.tweet-container > .tweet .pd-md .b-b-thin {
+  border: solid 5px lightgrey;
+}
+
+.tweet-container {
+  border: 3px solid lightgrey;
+  border-radius: 5px;
+}
+
+.right-title {
+  margin-top: 20px;
+  margin-left: 20px;
+}
+
+.right-side {
+  width: 33rem;
+  padding: 0.6rem 1.2rem 0rem 1.2rem;
+  margin-bottom: 1.2rem;
+}
+
+.right-side .searchbar {
+  position: relative;
+}
+
+.right-side .searchbar i {
+  position: absolute;
+  font-size: 2rem;
+  left: 24px;
+  top: 50%;
+  transform: translateY(-50%);
+  color: #a6b0b6;
+}
+
+.right-side .searchbar .search {
+  width: 100%;
+  padding: 1.6rem 2rem 1.6rem 6rem;
+  background-color: #eff3f4;
+  border: none;
+  outline: none;
+  border-radius: 20rem;
+  margin-top: 5px;
+}
+
+.right-side .searchbar .search::-moz-placeholder {
+  font-size: 1.5rem;
+}
+
+.right-side .searchbar .search:-ms-input-placeholder {
+  font-size: 1.5rem;
+}
+
+.right-side .searchbar .search::placeholder {
+  font-size: 1.5rem;
+}
+
+.right-side .searchbar .search:focus {
+  background-color: #fff;
+  border: 1px solid #4ba0eb;
+}
+
+.right-side .searchbar input[type=text] {
+  font-size: 1.5rem;
+  font-family: "Roboto", sans-serif;
+  font-weight: 300;
+}
+
+.right-side .news {
+  margin-top: 1.5rem;
+  background-color: #f7f9f9;
+  border-radius: 12px;
+}
+
+.right-side .news .title {
+  padding: 1rem;
+}
+
+.right-side .news-card {
+  padding: 1.2rem 1.6rem;
+  display: grid;
+  grid-template-columns: repeat(2, auto);
+  -moz-column-gap: 1rem;
+       column-gap: 1rem;
+}
+
+.right-side .news-card:hover {
+  background-color: rgba(180, 180, 180, 0.253);
+  transition: 0.3s all;
+  cursor: pointer;
+}
+
+.right-side .news-card-left {
+  grid-row: 1/4;
+  grid-column: 2/3;
+  justify-self: flex-end;
+  display: flex;
+  align-items: start;
+}
+
+.right-side .news-card-left img {
+  margin: 0 auto;
+  width: 7rem;
+  height: 90%;
+  -o-object-fit: cover;
+     object-fit: cover;
+  border-radius: 15px;
+}
+
+@media only screen and (max-width: 1050px) {
+  .right-side .news-card-left img {
+    height: 60%;
+  }
+}
+
+.right-side .card-bottom-text {
+  color: #4ba0eb;
+  padding: 1.4rem;
+}
+
+.right-side .card-bottom-text:hover {
+  background-color: rgba(180, 180, 180, 0.253);
+  transition: 0.3s all;
+  cursor: pointer;
+}
+
+.right-side .links ul {
+  display: flex;
+  flex-wrap: wrap;
+  -moz-column-gap: 0.8rem;
+       column-gap: 0.8rem;
+  margin: 1rem 0rem;
+}
+
+.right-side .links ul li {
+  list-style-type: none;
+  line-height: 2rem;
+}
+
+.right-side .links ul li a {
+  text-decoration: none;
+}
+
+.right-side .links ul li a:hover {
+  text-decoration: underline;
+  -webkit-text-decoration-color: #4ba0eb;
+          text-decoration-color: #4ba0eb;
+}
```

Comment:
Also, I kept all the code lines related to : hover at the top of the page together. It does not make any difference in the outcome but it makes it easier to find them if you need to work on the hover options.